app/scripts/components/Chart/chart-template.html

```diff
@@ -0,0 +1,3 @@
+<div class="c-chart">
+  <div class="chart-container" :ref="name"></div>
```

Comment:
`name` isn't the best name to follow in the code. Wouldn't be `chart` better that `name`?